SmartDeviceLink/SDLSecurityType.h

```diff
@@ -12,6 +12,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol SDLSecurityType <NSObject>
 
+@property (copy, nonatomic) NSString* appId;
```

Comment:
This also needs to be changed in https://github.com/smartdevicelink/sdl_security_ios